modules/get-started/pages/quick-start.adoc

```diff
@@ -200,10 +402,11 @@ Your message is displayed along with its metadata:
   "offset": 0
 }
 ```
-// (step {"action":"runShell", "command": "rpk topic consume chat-room --num 1", "output": "/Pandas are fabulous!/", "timeout": 10000})
+// (step {"action":"runShell", "command": "rpk topic consume chat-room --num 1 -X user=superuser -X pass=secretpassword", "output": "/Pandas are fabulous!/", "timeout": 10000})
 
+== Connect to the cluster externally
 
-To test external connectivity using your local machine:
+It's often more practical to connect to a remote cluster from your local machine with `rpk`. The `docker-compose.yml` file configured the containers to expose ports on your localhost, so you can communicate with the cluster outside the Docker network. To do so, download `rpk` on your local machine and configure it to connect to your cluster on the exposed ports.
 
 . Install rpk on your local machine:
 +
```

Comment:
just fyi, again: the stickiness between Linux/macOS tabs is annoying.

Reply:
Also I couldn't do this. Running on PC with WSL/Ubuntu/amd64. I'll try again tomorrow, start fresh.

Reply:
my stickiness comment stay, but with my fresh install today it works fine

Reply:
Fixed in https://github.com/redpanda-data/docs-ui/pull/246

When that PR is approved and merged, I'll release a new version of the UI.